test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -45,12 +45,20 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string DesktopRunnerTargetRuntime = "win7-x64";
     public const string CoreRunnerTargetRuntime = "";
     public const string InIsolation = "/InIsolation";
-
+    
     public const string NETFX452_48 = "net452;net461;net472;net48";
     public const string NETFX451_48 = "net452;net461;net472;net48";
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net451";
+    /// <summary>
+    /// Our current defaults for .NET and .NET Framework.
+    /// </summary>
+    public const string DEFAULT_RUNNER_NETFX_AND_NET = $"{DEFAULT_RUNNER_NETFX};netcoreapp2.1"; 
+    public const string DEFAULT_HOST_NETFX_AND_NET = "net451;netcoreapp2.1";
+    public const string LATEST_LEGACY = "Latest;LatestStable;LatestPreview;MostDownloaded;PreviousStable;LegacyStable";
+    public const string LATESTSTABLE_LEGACY = "LatestStable;LatestPreview;MostDownloaded;PreviousStable;LegacyStable";
```

Comment:
Is the intent of this one to be the same as previous without preview? Currently it's the same as previous except for `Latest` which I am not sure to understand what it means.

Reply:
The intent was to have the same without Latest. That is published preview and release versions, which is what makes sense for mstest at the moment, because we don't build it in this pipeline. But maybe I can later grab the latest version of mstest from a public feed automatically and use that.